app/src/main/java/com/github/onlynotesswent/model/folder/FolderViewModel.kt

```diff
@@ -67,6 +67,15 @@ class FolderViewModel(private val repository: FolderRepository) : ViewModel() {
   fun selectedFolder(folder: Folder) {
     _selectedFolder.value = folder
   }
+  /**
```

Comment:
Good addition of functions to handle folders and subfolders states. However, what do you think of modifying selected folder to accept a nullable argument so that we can directly set it to null by calling `selectedFolder(null)` instead of using clearSelectedFolder

Reply:
I think it is cleaner to have a clearFunction. It would seem a bit weird I feel like to call selectedFolder(null). When reading the code I feel like having a explicit clearSelectedFolder method to be better.